src/Http/Http.Results/src/AcceptedAtRoute.cs

```diff
@@ -53,7 +53,7 @@ internal AcceptedAtRoute(string? routeName, object? routeValues)
         RouteValueDictionary routeValues)
     {
         RouteName = routeName;
-        RouteValues = routeValues;
+        RouteValues = routeValues ?? new RouteValueDictionary();
```

Comment:
`routeValues` is not declared as nullable, and this is an internal method. I don't think we need to check it for `null`. Anyone passing a nullable routeValues will get a warning. This shouldn't be needed.

Either that, or the signature of this method should be updated to `RouteValueDictionary? routeValues`.

Same comment applies below.